extensions/ql-vscode/src/pure/log-summary-parser.ts

```diff
@@ -0,0 +1,51 @@
+// REVIEW: Perhaps this file should be renamed now that it also includes the 
+// data model on top of the parsing logic?
+
+// REVIEW: Suggestions on other fields that are useful for a performance 
+// debugger would be welcome!
+export interface EvaluatorLogData {
```

Comment:
One suggestion here: it might make your life easier to have the field names exactly match the fields from the evaluator log. Then it's easy to update this file (we could even derive it in future from the JSON schema, but that's a future idea).

Reply:
That makes sense — I think I'd named them whatever was intuitive to me, but it'd be better to keep it consistent with the actual log for future maintenance. Deriving it from a JSON schema certainly seems like it'd make the parsing logic less brittle. I'll update it now to match the evaluator log fields.